lib/mayaUsd/ufe/UsdTRSUndoableCommandBase.h

```diff
@@ -80,24 +78,29 @@ class MAYAUSD_CORE_PUBLIC UsdTRSUndoableCommandBase
     virtual void addEmptyAttribute() = 0;
     virtual bool cannotInit() const;
 
+    // Create a UsdSceneItem::Ptr conditionaly in the first access from Ufe::Path.
+    void conditionalCreateItem();
```

Comment:
Make this const and make fItem mutable.

Reply:
Maybe conditionalCreateItem() is wordy?  How about updateItem()?

Reply:
@ppt-adsk  Good point. makes sense. Should I have done it is the first place . 

addressed in 07014386de044e301769584c08bdb7094844b30f